google-cloud-storage/src/main/scala/akka/stream/alpakka/googlecloud/storage/StorageObject.scala

```diff
@@ -380,15 +380,15 @@ object StorageObject {
       timeCreated: OffsetDateTime,
       timeDeleted: Option[OffsetDateTime],
       storageClass: String,
-      contentDisposition: String,
-      contentEncoding: String,
-      contentLanguage: String,
+      contentDisposition: Option[String],
```

Comment:
Should this not be Optional or just String and accept null?

Reply:
@ennru do you have some guidance here on what I should do for the java api?

Reply:
If this is indeed an optional parameter then use `Option`.

Reply:
See earlier comment about adding a Java API.

Reply:
@seglo  This method is marked as java api, I would expect nullable values here or Optional